### PR TITLE
Updating NW.js and ffmpeg libraries to new 0.47

### PR DESCRIPTION
**Pull request by Asizon**
Seems that finnaly we have available the latest version with ffmpeg compatibility, I have updated both to 0.47.0.

### CODE REVIEW

**Comment by Asizon** [comment]
Now we have all ffmpeg libraries updated to 0.47 to go with NWjs 47.0, verified video codecs and seems that it works as expected.

**Comment by Asizon** [comment]
Also export video feature works as expected:

image

https://github.com/betaflight/blackbox-log-viewer/pull/456 needed to work

File: library/Readme.md [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 ### ffmpeg
 
-Current version: 0.44.6
+Current version: 0.47.0
 
-Updating: Download prebuilt libraries from https://github.com/iteufel/nwjs-ffmpeg-prebuilt/releases.
+Updating: Download prebuilt libraries from https://github.com/iteufel/nwjs-ffmpeg-prebuilt/releases
```
Comment (mikeller) [inline library/Readme.md:7]: Why are you removing the punctuation from this sentence? ;-)

Comment (Asizon) [reply library/Readme.md:7]: @mikeller I think that this punctuation is breaking the link when you tried to open it and is better to remove it, what do you think?

Comment (mikeller) [reply library/Readme.md:7]: Weird - this is working fine for me here - what operating system / browser do you have the problem with?

Comment (Asizon) [reply library/Readme.md]: ups sorry, tested it now in google chrome and works, maybe i have tested it directly from Notepad++ not sure, fixed!

**Pull request merged**